cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -1208,19 +1214,235 @@ instance (ToObject peer) => ToObject (WithMuxBearer peer MuxTrace) where
+    mkObject [ "kind" .= String "LocalRootWaiting"
+             , "domainAddress" .= show d
+             , "diffTime" .= show dt
+             ]
+  toObject _verb (TraceLocalRootResult d res) =
+    mkObject [ "kind" .= String "LocalRootResult"
+             , "domainAddress" .= show d
+             , "result" .= show res
+             ]
+  toObject _verb (TraceLocalRootGroups groups) =
+    mkObject [ "kind" .= String "LocalRootGroups"
+             , "localRootGroups" .= show groups
+             ]
+  toObject _verb (TraceLocalRootFailure d dexception) =
+    mkObject [ "kind" .= String "LocalRootFailure"
+             , "domainAddress" .= show d
+             , "error" .= show dexception
+             ]
 
 instance ToObject TracePublicRootPeers where
-  toObject _verb ev =
-    mkObject [ "kind" .= String "TracePublicRootPeers"
-             , "event" .= show ev ]
+  toObject _verb (TracePublicRootRelayAddresses l_r) =
+    mkObject [ "kind" .= String "PublicRootRelayAddresses"
+             , "relayAddresses" .= show l_r
+             ]
+  toObject _verb (TracePublicRootDomains l_d) =
+    mkObject [ "kind" .= String "PublicRootDomains"
+             , "domainAddresses" .= show l_d
+             ]
+  toObject _verb (TracePublicRootResult b res) =
+    mkObject [ "kind" .= String "PublicRootResult"
+             , "domain" .= show b
+             , "result" .= show res
+             ]
+  toObject _verb (TracePublicRootFailure b d) =
+    mkObject [ "kind" .= String "PublicRootFailure"
+             , "domain" .= show b
+             , "error" .= show d
+             ]
+
+instance ToJSON PeerStatus where
+  toJSON PeerCold =
+    Aeson.object [ "kind" .= String "PeerStatus"
+                 , "status" .= String "PeerCold"
+                 ]
+  toJSON PeerWarm =
+    Aeson.object [ "kind" .= String "PeerStatus"
+                 , "status" .= String "PeerWarm"
+                 ]
+  toJSON PeerHot =
+    Aeson.object [ "kind" .= String "PeerStatus"
+                 , "status" .= String "PeerHot"
+                 ]
+
+instance Aeson.ToJSONKey Socket.SockAddr where
+
+instance (Aeson.ToJSONKey peerAddr, Show peerAddr)
+  => ToJSON (LocalRootPeers peerAddr) where
+  toJSON lrp =
+    Aeson.object [ "kind" .= String "LocalRootPeers"
+                 , "state" .= toJSON (toMap lrp)
+                 , "groups" .= show (toGroupSets lrp)
+                 ]
+
+instance ToJSON PeerSelectionTargets where
+  toJSON (PeerSelectionTargets i i2 i3 i4) =
+    Aeson.object [ "kind" .= String "PeerSelectionTargets"
+                 , "nRootPeers" .= i
+                 , "nKnownPeers" .= i2
+                 , "nEstablishedPeers" .= i3
+                 , "nActivePeers" .= i4
+                 ]
 
 instance ToObject (TracePeerSelection Socket.SockAddr) where
-  toObject _verb ev =
-    mkObject [ "kind" .= String "TracePeerSelection"
-             , "event" .= show ev ]
+  toObject _verb (TraceLocalRootPeersChanged ls ls3) =
```

Comment:
Why `ls3` (does the 3 signifies something)?  Why not just `ls'`?

The same applies to other names in this module.

Reply:
Yeah the names were generated by HLS Wingman feature, I'll make those better!